<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>create_vertex_dialog</class>
 <widget class="QDialog" name="create_vertex_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>64</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Create Vertex</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_xPos">
     <property name="text">
      <string>X-Position</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_yPos">
     <property name="text">
      <string>Y-Position</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_label">
     <property name="text">
      <string>Label</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_connectTo">
     <property name="text">
      <string>Connect Label</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="edit_xPos"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="edit_yPos"/>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="edit_label"/>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="edit_connectTo"/>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="button_add">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
